README.md

```diff
@@ -0,0 +1,248 @@
+[![openssl logo][]][www.openssl.org]
+
+[![travis badge][]][travis jobs]
+[![appveyor badge][]][appveyor jobs]
+
+
+
+Welcome to the OpenSSL Project
+==============================
+
+OpenSSL is a robust, commercial-grade, full-featured Open Source Toolkit
+for the Transport Layer Security (TLS) protocol formerly known as the
+Secure Sockets Layer (SSL) protocol. The protocol implementation is based
+on a full-strength general purpose cryptographic library, which can also
+be used stand-alone.
+
+OpenSSL is descended from the SSLeay library developed by Eric A. Young
+and Tim J. Hudson.
+
+The official Home Page of the OpenSSL Project is [www.openssl.org][].
+
+
+Table of Contents
+=================
+
+ - [Overview](#overview)
+ - [Download](#download)
+ - [Build and Install](#build-and-install)
+ - [Documentation](#documentation)
+ - [License](#license)
+ - [Support](#support)
+ - [Contributing](#contributing)
+ - [Legalities](#legalities)
+
+Overview
+========
+
+The OpenSSL toolkit includes:
+
+- **libssl**
+  an implementation of all TLS protocol versions up to TLSv1.3 ([RFC 8446][]).
+  The older and now insecure SSL protocol versions (SSLv2 and SSLv3) are still
```

Comment:
SSLv2 is not supported as 1.0.2 is EOL (I do not think we need to advertise the extended support for it in this way - hopefully nobody really uses 1.0.2 because it supports SSLv2).

Reply:
Well, let's cut off the old braids then (as we say in German), see b5c854633b673704a0542648fa4f9abe44ac8727.